src/app.py

```diff
@@ -7,7 +7,6 @@
     # additional_projections_chart,
```

Comment:
do me a favor and remove this commented out import please?

Reply:
👍 done